### PR TITLE
Indicate metrics are reset upon restart and mark the ones the don't

### PR DESCRIPTION
**Pull request by asafm**
Topic Stats metrics are mainly reset to 0 when the broker restarts, aside from a few gauges which do not reset as they take their values from persistent storage (BK).

This is now indicated to prevent confusion for users using it.

### CODE REVIEW

File: docs/administration-stats.md [hunk 1]
```diff
@@ -8,32 +8,39 @@ sidebar_label: "Pulsar statistics"
...
-| filteredEntriesCount                             | The count of skipped entries for the topic.                                                              |
-| nonContiguousDeletedMessagesRanges               | The number of non-contiguous deleted messages ranges.                                                    |
-| nonContiguousDeletedMessagesRangesSerializedSize | The serialized size of non-contiguous deleted messages ranges.                                           |
-| ownerBroker                                      | The broker that owns this topic.                                                                         |
-| [publishers](#producer-stats)                    | The list of all local publishers on the topic, ranging from zero to thousands.                           |
-| [subscriptions](#subscription-stats)             | The list of all local subscriptions to the topic.                                                        |
-| [replication](#replication-stats)                | This section gives the stats for cross-colo replication of this topic.                                   |
+:::note
```
Comment (asafm) [inline docs/administration-stats.md:11]: image

File: docs/administration-stats.md [hunk 2]
```diff
@@ -8,32 +8,39 @@ sidebar_label: "Pulsar statistics"
...
+
+All metrics below are *reset* to 0 upon broker restart, *except* gauges marked as "Not reset upon restart".
```
Comment (asafm) [inline docs/administration-stats.md:13]: image

File: docs/administration-stats.md [hunk 3]
```diff
@@ -8,32 +8,39 @@ sidebar_label: "Pulsar statistics"
...
+
```
Comment (asafm) [inline docs/administration-stats.md:14]: image

File: docs/administration-stats.md [hunk 4]
```diff
@@ -67,7 +74,7 @@ The following table outlines the stats of a topic. For more details about how to
...
 | chunkedMessageRate                               | The chunked message dispatch rate.                                                                                                                                                                                                                                   |
-| earliestMsgPublishTimeInBacklog                  | The publish time of the earliest message in the backlog for the subscription (in milliseconds).                                                                                                                                                                      |
+| earliestMsgPublishTimeInBacklog                  | The publish time of the earliest message in the backlog for the subscription (in milliseconds). Not reset upon restart                                                                                                                                               |
```
Comment (asafm) [inline docs/administration-stats.md:77]: image

File: versioned_docs/version-2.10.x/admin-api-topics.md [hunk 5]
```diff
@@ -290,7 +290,13 @@ admin.topics().unload(topic);
 
 You can check the following statistics of a given non-partitioned topic.
 
-  -   **msgRateIn**: The sum of all local and replication publishers' publish rates (msg/s).
+:::note
+
+All metrics below are *reset* to 0 upon broker restart, *except* gauges marked as "Not reset upon restart".
```
Comment (asafm) [inline versioned_docs/version-2.10.x/admin-api-topics.md:295]: image

File: versioned_docs/version-2.10.x/admin-api-topics.md [hunk 6]
```diff
@@ -300,7 +306,7 @@ You can check the following statistics of a given non-partitioned topic.
 
   -   **averageMsgSize**: The average size (in bytes) of messages published within the last interval.
 
-  -   **storageSize**: The sum of the ledgers' storage size for this topic. The space used to store the messages for the topic.
+  -   **storageSize**: The sum of the ledgers' storage size for this topic. The space used to store the messages for the topic. Not reset upon restart.
```
Comment (asafm) [inline versioned_docs/version-2.10.x/admin-api-topics.md:309]: image

File: versioned_docs/version-2.10.x/admin-api-topics.md [hunk 7]
```diff
@@ -314,9 +320,9 @@ You can check the following statistics of a given non-partitioned topic.
 
   -   **msgChunkPublished**: Topic has chunked message published on it.
 
-  -   **backlogSize**: Estimated total unconsumed or backlog size (in bytes).
+  -   **backlogSize**: Estimated total unconsumed or backlog size (in bytes). Not reset upon restart.
```
Comment (asafm) [inline versioned_docs/version-2.10.x/admin-api-topics.md:323]: image

File: versioned_docs/version-2.10.x/admin-api-topics.md [hunk 8]
```diff
@@ -384,7 +390,7 @@ You can check the following statistics of a given non-partitioned topic.
 
           -   **backlogSize**: Size of backlog for this subscription (in bytes).
           
-          -   **earliestMsgPublishTimeInBacklog**: The publish time of the earliest message in the backlog for the subscription (ms).
+          -   **earliestMsgPublishTimeInBacklog**: The publish time of the earliest message in the backlog for the subscription (ms). Not reset upon restart.
```
Comment (asafm) [inline versioned_docs/version-2.10.x/admin-api-topics.md:393]: image

File: docs/administration-stats.md (hunk 4 above)
Comment (Anonymitaet) [inline docs/administration-stats.md:77]:
```suggestion
| earliestMsgPublishTimeInBacklog                  | The publish time of the earliest message in the backlog for the subscription (in milliseconds). Note that **this metric is not reset upon restart**.                                                                                                                                               |
```

File: docs/administration-stats.md [hunk 9]
```diff
@@ -86,7 +93,7 @@ The following table outlines the stats of a topic. For more details about how to
...
 | msgOutCounter                                    | The total messages delivered to a consumer.                                                                                                                                                                                                                          |
-| backlogSize                                      | The size of backlog for this subscription (in bytes).                                                                                                                                                                                                                |
+| backlogSize                                      | The size of backlog for this subscription (in bytes). Not reset upon restart                                                                                                                                                                                         |
```
Comment (Anonymitaet) [inline docs/administration-stats.md:96]:
```suggestion
| backlogSize                                      | The size of backlog for this subscription (in bytes). Note that **this metric is not reset upon restart**.                                                                                                                                                                                          |
```

File: versioned_docs/version-2.10.x/admin-api-topics.md (hunk 7 above)
Comment (Anonymitaet) [inline versioned_docs/version-2.10.x/admin-api-topics.md:323]:
```suggestion
  -   **backlogSize**: Estimated total unconsumed or backlog size (in bytes). Note that **this metric is not reset upon restart**. 
```

File: versioned_docs/version-2.10.x/admin-api-topics.md [hunk 10]
```diff
@@ -314,9 +320,9 @@ You can check the following statistics of a given non-partitioned topic.
 
   -   **msgChunkPublished**: Topic has chunked message published on it.
 
-  -   **backlogSize**: Estimated total unconsumed or backlog size (in bytes).
+  -   **backlogSize**: Estimated total unconsumed or backlog size (in bytes). Not reset upon restart.
 
-  -   **offloadedStorageSize**: Space used to store the offloaded messages for the topic (in bytes).
+  -   **offloadedStorageSize**: Space used to store the offloaded messages for the topic (in bytes). Not reset upon restart.
```
Comment (Anonymitaet) [inline versioned_docs/version-2.10.x/admin-api-topics.md:325]:
```suggestion
  -   **offloadedStorageSize**: Space used to store the offloaded messages for the topic (in bytes). Note that **this metric is not reset upon restart**. 
```

File: versioned_docs/version-2.10.x/admin-api-topics.md (hunk 8 above)
Comment (Anonymitaet) [inline versioned_docs/version-2.10.x/admin-api-topics.md:393]:
```suggestion
          -   **earliestMsgPublishTimeInBacklog**: The publish time of the earliest message in the backlog for the subscription (ms). Note that **this metric is not reset upon restart**. 
```

File: contribute/document-preview.md [hunk 11]
```diff
@@ -60,6 +60,7 @@ By default, a browser window will open at [http://localhost:3000](http://localho
 
 ![alt_text](media/website-preview.png)
 
+When you click on Docs, you will be taken to the latest stable version (e.g. `http://localhost:3000/docs/2.10.x/`).  If you're previewing `master` which is current, then you need to change the url to `http://localhost:3000/docs/next`
```
Comment (Anonymitaet) [inline contribute/document-preview.md:63]:
```suggestion
::: tip
When you click on `Docs`, you are taken to the latest stable version (e.g., `http://localhost:3000/docs/2.10.x/`). If you want to preview changes on `master`, change the URL to `http://localhost:3000/docs/next`
:::
```
Write in the simple present tense as much as possible if you are covering facts that were, are, and forever shall be true. https://docs.google.com/document/d/1lc5j4RtuLIzlEYCBo97AC8-U_3Erzs_lxpkDuseU0n4/edit?pli=1#bookmark=id.e8uqh1awkcnp

**Review by Demogorgon314 — APPROVED** [review]
LGTM.

**Review by Anonymitaet — APPROVED** [review]

**Pull request merged**